tablestore/db_test.go

```diff
@@ -0,0 +1,52 @@
+package tablestore
+
+import (
+    "testing"
+    "github.com/google/uuid"
+    "fmt"
+)
+
+func TestSave(t *testing.T) {
```

Comment:
Make sure to actually assert your tests for expected output, not just print to console